vtysh/bgp_vty.c

```diff
@@ -4762,12 +4762,12 @@ cli_bgp_neighbor_weight_execute (char *vrf_name,
 
     if (!weight_str || !set) {
         weight = BGP_ATTR_DEFAULT_WEIGHT;
+        ovsrec_bgp_neighbor_set_weight (bgp_neighbor_row, &weight, 0);
```

Comment:
1) `weight = BGP_ATTR_DEFAULT_WEIGHT;` is unnecessary, I forgot to delete it, will change  
2) There is no unset operation for columns in OVSDB, so `ovsrec_bgp_neighbor_set_weight` with 3rd parameter `size_t n_weight` set to 0 must be called to remove value from column weight in OVSDB.